include/file.iol

```diff
@@ -77,6 +77,18 @@ type ListResponse:void {
 interface FileInterface {
 RequestResponse:
 	/**!
+	 * Constructs an absolute path to the target file or directory. 
+	 * Can be used to construct an absolute path for new files that does not exist yet.
+	 */
+	getAbsolutePath( string )( string ),
+
+	/**!
+	 * Constructs an absolute path to the parent directory of the target file or directory.
+	 * Can be used to construct an absolute path to the parent directory for files that does not exists yet.
+	 */
+	getAbsoluteParentPath( string )( string ),
```

Comment:
Why not simply getParentPath ? Then using this becomes a matter of:

toAbsolutePath@File( path )( path );
getParentPath@File( path )( path )